charts/uptime-kuma/Chart.yaml

```diff
@@ -11,4 +11,4 @@ name: uptime-kuma
 sources:
 - https://github.com/louislam/uptime-kuma
 type: application
-version: 0.3.7
+version: 0.4.0
```

Comment:
Your thoughts on the Version change are valid.
The Statefulset causes a new creation of a PersistentVolume in Kubernetes.

Therefore we should bump the version to 1.0.0 to reflect this major change.

Reply:
done!